cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -1290,7 +1322,8 @@ importExpr' sc env schema expr =
     C.ELocated _ e ->
       importExpr' sc env schema e
 
-    C.ECase {} -> panic "importExpr" ["`case` is not yet supported"]
+    C.ECase {} -> fallback
+                  -- FIXME: Not sure using fallback is correct here.
```

Comment:
I think `fallback` is correct choice here. Per the Haddocks, you'd use `importExpr` when the type must be inferred, and that is indeed the case for Cryptol `case` expressions.